src/mixins/createProjectorMixin.ts

```diff
@@ -142,6 +143,10 @@ function attach(instance: Projector, { type }: AttachOptions) {
 
 	projectorData.attachPromise = new Promise((resolve, reject) => {
 		projectorData.afterCreate = () => {
+			if (projectorData.originalAfterCreate) {
+				projectorData.originalAfterCreate.call(instance);
```

Comment:
in fact, it should perhaps be called on the scope of whatever the `bind` property passed to maquette is I think?